### PR TITLE
Updated Rinkeby -> Goerli Metamask Screenshot

### PR DESCRIPTION
**Pull request by brandonfrulla**

### CODE REVIEW

**Comment by brandonfrulla** [comment]
Hey there, I updated the imugr screenshot to point to https://imgur.com/a/j8iN46I instead of the old Metamask screenshot that showed being connected to Rinkeby. Figured newer people would get confused by the discrepancy

**Comment by brandonfrulla** [comment]
@AlmostEfficient Pls review when you get a chance :)

File: NFT_Game/en/Section_1/Lesson_5_Deploy_To_Goerli.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ In order get fake ETH, we have to ask the network for some. **This fake ETH wil
 
 Please make sure you're on the **Goerli** network on Metamask. This is a super common issue I see!
 
-![Untitled](https://i.imgur.com/MKjsJXX.png)
+![Untitled](https://imgur.com/a/j8iN46I)
```
Comment (helloitsm3) [inline NFT_Game/en/Section_1/Lesson_5_Deploy_To_Goerli.md:66]: The image link seems to be broken. Please use a raw image link. Update it to using `https://i.imgur.com/cI4a1Oh_d.webp?maxwidth=760&fidelity=grand`

Comment (brandonfrulla) [reply NFT_Game/en/Section_1/Lesson_5_Deploy_To_Goerli.md:66]: Understood-- updating now!

**Review by AlmostEfficient — APPROVED** [review]

**Pull request merged**